models.py

```diff
@@ -43,6 +48,21 @@ def json(self):
     def verify_password(self, origin_password):
         return pbkdf2_sha256.verify(origin_password, self.password)
 
+    def favorite_items(self):
+        return [favorite.item.json() for favorite in self.favorites]
+
+    def add_favorite(self, item):
+        favorite = Favorites.create(
+            uuid=uuid.uuid4(),
+            user=self,
+            item=item,
+        )
+        return favorite
+
+    def remove_favorite(self, item):
+        Favorites.delete().where(Favorites.item == item).where(Favorites.user == self).execute()
```

Comment:
potevi mettere entrambe le condizioni in un unico where separato da una virgola, come mostrato qua:
```
http://docs.peewee-orm.com/en/latest/peewee/querying.html#filtering-records
```